tmt/convert.py

```diff
@@ -516,7 +552,14 @@ def target_content(target):
     return common_data, individual_data
 
 
-def read_nitrate(beaker_task, common_data, disabled, general):
+def read_nitrate(beaker_task: str,
+                 common_data: Dict[str,
+                                   Any],
+                 disabled: bool,
+                 general: bool) -> Tuple[Dict[str,
```

Comment:
@ukulekek please create an alias for this crazy return type

Reply:
Created